experimental/generation/generator/templates/library-CompleteConfirmation.dialog.lg

```diff
@@ -10,11 +10,11 @@
     "actions": [
         {
             "$kind": "Microsoft.SendActivity",
-            "activity": "\${showForm('${prefix}', publicProperties())}"
+            "activity": "\${getFormAsActivity('${prefix}', getPublicPropertiesText())}"
```

Comment:
>Form [](start = 31, length = 4)

Not sure if we should still keep form as a name, we don't use it anywhere else.  Not sure what would be better though.  'result'?  'schema'?

Reply:
dialog?


---
In reply to: [433372123](https://github.com/microsoft/BotBuilder-Samples/pull/2461#discussion_r433372123) [](ancestors = 433372123)